src/popup/warnings/util.js

```diff
@@ -0,0 +1,3 @@
+export function hasText (warningStyle) {
+  return ['bottomBanner', 'topBanner'].includes(warningStyle)
+}
```

Comment:
This function really needs to go. It made sense back when there was only "text" as a type of field but now that we have colors too, setting things up like that is going to be unmanageable.